pwr.md

```diff
@@ -0,0 +1,144 @@
+# pwr module
+
+This is a python module which allows an application to modify power attributes of a CPU. Manipulation can be done to the frequency of cores, 
```

Comment:
I initially wanted to suggest using doxygen to document the library, but Doxygen actually poorly supports Python code and doesn't have a native way to document object attributes (only methods). So, disregard this :D